control-plane/drizzle/0019_early_silver_sable.sql

```diff
@@ -0,0 +1,4 @@
+DROP TABLE "users";--> statement-breakpoint
```

Comment:
Assuming this just wasn't committed previously?

Reply:
Yes. Probably. Thanks!